guides/Getting Started.md

```diff
@@ -0,0 +1,696 @@
+
+```elixir
+defmodule Friends.Repo.Migrations.CreatePeople do
+  use Ecto.Migration
+
+  def change do
+    create table(:people) do
+      add :first_name, :string
+      add :last_name, :string
+      add :age, :integer
+    end
+  end
+end
+```
+
+This new code will tell Ecto to create a new table called people, and add three new fields: `first_name`, `last_name` and `age` to that table. The types of these fields are `string` and `integer`. (The different types that Ecto supports are covered in the [Ecto.Schema](https://hexdocs.pm/ecto/Ecto.Schema.html) documentation.)
+
+**NOTE: The naming convention for tables in Ecto databases is to use a pluralized name.**
+
+To run this migration and create the `people` table in our database, we will run this command:
+
+```
+mix ecto.migrate
+```
+
+If we found out that we made a mistake in this migration, we could run `mix ecto.rollback` to undo the changes in the migration. We could then fix the changes in the migration and run `mix ecto.migrate` again. If we ran `mix ecto.rollback` now, it would delete the table that we just created.
+
+We now have a table created in our database. The next step that we'll need to do is to create the schema.
+
+## Creating the schema
+
+The schema is an Elixir representation of data from our database. Each schema doesn't necessarily need to be tied to a table; they can be tied to database views too.
+
+Let's create the schema within our application at `lib/friends/person.ex`:
+
+```elixir
+defmodule Friends.Person do
+  use Ecto.Schema
+
+  schema "people" do
+    field :first_name, :string
+    field :last_name, :string
+    field :age, :integer
+  end
+end
+```
+
+This model defines the schema from the database that this model maps to. In this case, we're telling Ecto that the `Friends.Person` model maps to the `people` table in the database, and the `first_name`, `last_name` and `age` fields in that table. The second argument passed to `field` tells Ecto how we want the information from the database to be represented in our model.
+
+**We've called this model `Person` because the naming convention in Ecto for models is a singularized name.**
+
+We can play around with this model in an IEx session by starting one up with `iex -S mix` and then running this code in it:
+
+```elixir
+person = %Friends.Person{}
+```
+
+This code will give us a new `Friends.Person` struct, which will have `nil` values for all the fields. We can set values on these fields by generating a new struct:
+
+```elixir
+person = %Friends.Person{age: 28}
+```
+
+Or with syntax like this:
+
+```elixir
+%{person | age: 28}
+```
+
+We can retrieve values using this syntax:
+
+```elixir
+person.age # => 28
+```
+
+Let's take a look at how we can insert data into the database.
+
+## Inserting data
+
+We can insert a new record into our `people` table with this code:
+
+```elixir
+person = %Friends.Person{}
+Friends.Repo.insert person
```

Comment:
I think parens are [recommended](https://github.com/niftyn8/elixir_style_guide) for all function calls.  This documentation would be a great place to set the example if that's correct.